scripts/tfc/training_data.py

```
@@ -0,0 +1,59 @@
+import pandas as pd
+from pathlib import Path 
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)